d2l-table-shared-styles.html

```diff
@@ -53,6 +53,7 @@
 			};
 			--d2l-table-row: {
 				display:table-row;
+				background-color: var(--d2l-table-body-background-color);
```

Comment:
I don't see this variable being set to anything anywhere, is it actually needed?